src/app/api/peanut/user/get-user-from-cookie/route.ts

```diff
@@ -1,16 +1,15 @@
 import { cookies } from 'next/headers'
 import { NextRequest, NextResponse } from 'next/server'
 
-export async function GET(request: NextRequest) {
+export async function GET(_request: NextRequest) {
     const cookieStore = cookies()
     const token = cookieStore.get('jwt-token')
 
     if (!token) {
         return new NextResponse('Bad Request: missing required parameters', { status: 400 })
     }
     try {
-        const { protocol, hostname, port } = new URL(request.url)
-        const apiUrl = `${protocol}//${hostname}:${port}/api/peanut/user/get-user`
+        const apiUrl = `${process.env.NEXT_PUBLIC_BASE_URL}/api/peanut/user/get-user`
 
         const response = await fetch(apiUrl, {
             method: 'POST',
```

Comment:
I'm confused, in our FE we're adding authToken in body, but in our API `verifyAuth` and `requireApiKey` should both fail for this request?

Reply:
Also, we have get-user and get-user-from-cookie routes - what's the difference?

In general i think  we'd benefit from cleaning up the plethora of routes we seem to have on the client backend

Reply:
`get-user` seems like it's not being used anywhere